annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/Config.java

```diff
@@ -72,4 +73,14 @@ public interface Config {
    */
   @Nonnull
   Path getOutputDirectory();
+
+  /**
+   * Returns source for the element at the given path. If the element exists in the source code
+   * {@code "SOURCE"} will be returned and otherwise the name of the code generator will be
+   * returned.
+   *
+   * @return Name of the generator that produced the code and {@code "SOURCE"} if element exists in
+   *     source code.
+   */
+  String getSourceForSymbolAtPath(TreePath path);
```

Comment:
Again use enum for return

Reply:
4bbd2e4c7539c60756522d7238db930d34bcd1d2